src/ADAL.PCL/Internal-Telemetry/EventConstants.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.IdentityModel.Clients.ActiveDirectory
+{
+    internal class EventConstants
+    {
+        public const string EventName = "event_name";
+        public const string ApplicationName = "application_name";
+        public const string ApplicationVersion = "application_version";
+        public const string SdkVersion = "sdk_version";
+        public const string SdkId = "sdk_id";
+        public const string DeviceId = "device_id";
+        public const string Tenant = "tenant";
+        public const string Issuer = "issuer";
+        public const string Idp = "idp";
+        public const string Upn = "upn";
+        public const string Email = "email";
+        public const string PasswordExpiration = "pwd_exp";
+        public const string PasswordChangeUrl = "pwd_url";
+        public const string FamilyName = "family_name";
+        public const string ResponseTime = "response_time";
+        public const string ClientIp = "client_ip";
+        public const string ClientId = "client_id";
+        public const string BrokerEvent = "BrokerEvent";
```

Comment:
Mixed cases ?

---
In reply to: [74547495](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r74547495) [](ancestors = 74547495)